src/schema/types/paths.ts

```diff
@@ -46,7 +46,9 @@ export type ItemSchemaPaths<SCHEMA extends ItemSchema = ItemSchema> = ItemSchema
       : never
     : never
 
-type AnySchemaPaths<SCHEMA_PATH extends string = ''> = `${SCHEMA_PATH}${string}`
+type AnySchemaPaths<SCHEMA_PATH extends string = ''> = SCHEMA_PATH extends ''
```

Comment:
I think you can remove the ternary because it breaks the default case. It makes sense to have: `AnySchemaPaths = string`